3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/fibonacci/src/lib.rs

```diff
@@ -1,10 +1,13 @@
 pub fn fib(n: u32) -> u32 {
     if n <= 2 {
-        // The base case
-        todo!("Implement this");
+        if n <= 0 {
+            return 0;
+        }
+        else{
+            return 1;
+        }
     } else {
-        // The recursive case
-        todo!("Implement this");
+        return fib(n-1) + fib(n-2);
```

Comment:
1. `return`을 생략할 수 있습니다.
2. `cargo fmt`를 통해 코드 스타일을 맞춰주세요.

```suggestion
        fib(n - 1) + fib(n - 2)
```